modules/custom/d_p/src/Plugin/Field/FieldWidget/SettingsWidget.php

```diff
@@ -488,4 +493,24 @@ public function validate($element, FormStateInterface $form_state) {
     $form_state->setValueForElement($element, json_encode($values));
   }
 
+  public static function getModifierDefaults() {
+    $modifiers = self::getConfigOptions()[self::CSS_CLASS_SETTING_NAME]['modifiers'];
+    $defaults = [];
+    foreach ($modifiers as $key => $modifier) {
+      if ($modifier['type'] === 'select') {
+        if ($modifier['default']) {
```

Comment:
ok, w większości przypadków nie ma klucza `'default'` w modyfikatorach tam, to nie przeszkadza, że odwołujesz się tu do klucza tabeli, który nie istnieje? nie lepiej sprawdzać to jakimś `isset` ?